### ISSUE TITLE
Generic "play the news" intent not working

### ISSUE THREAD

**Issue by ChanceNCounter** [MycroftAI/skill-npr-news#59]
I've pulled the commit just PRed to the marketplace, and the generic "play the news" utterance does not default to NPR. "Play NPR" works just fine, but, "play the news" throws this:

```
>> Here is the latest news from Your custom feed.
>> There seems to be a technical issue, I couldn't start that news feed.
```

```
2020-01-17 01:36:15.505 | INFO     | 15524 | Playback Control Skill | Resolving Player for: the news
Removing event mycroft-playback-control.mycroftai:PlayQueryTimeout
Removing event mycroft-playback-control.mycroftai:PlayQueryTimeout
2020-01-17 01:36:20.057 | INFO     | 15524 | Playback Control Skill | Playing with: skill-npr-news
2020-01-17 01:36:20.217 | ERROR    | 15524 | NewsSkill | Error: 'NoneType' object is not subscriptable
2020-01-17 01:36:20.219 | INFO     | 15524 | NewsSkill | Traceback: Traceback (most recent call last):
  File "/opt/mycroft/skills/skill-npr-news/__init__.py", line 253, in handle_latest_news
    url = self.get_media_url(rss)
  File "/opt/mycroft/skills/skill-npr-news/__init__.py", line 210, in get_media_url
    if station_url[-4:] in DIRECT_PLAY_FILETYPES:
TypeError: 'NoneType' object is not subscriptable
```

**Comment by forslund** [comment]
Thanks for reporting @ChanceNCounter will look into it directly. Marking the PR on mycroft-skills as on hold

**Comment by forslund** [comment]
Should be resolved through #60. let us know if you find more issues.
